ubiquibot-config-default.yml

```
@@ -0,0 +1,41 @@
+---
+chain-id: 1
+base-multiplier: 1000
+issue-creator-multiplier: 2000
+time-labels:
+  - name: "Time: <1 Hour"
+    weight: 0.125
+    value: 3600
+  - name: "Time: <1 Day"
+    weight: 1
+    value: 86400
+  - name: "Time: <1 Week"
+    weight: 2
+    value: 604800
+  - name: "Time: <2 Weeks"
+    weight: 3
+    value: 1209600
+  - name: "Time: <1 Month"
+    weight: 4
+    value: 2592000
+priority-labels:
+  - name: "Priority: 0 (Normal)"
+    weight: 1
+  - name: "Priority: 1 (Medium)"
+    weight: 2
+  - name: "Priority: 2 (High)"
+    weight: 3
+  - name: "Priority: 3 (Urgent)"
+    weight: 4
+  - name: "Priority: 4 (Emergency)"
+    weight: 5
+auto-pay-mode: true
```

Maybe its more clear to say the following:

```suggestion
payment-permit-generation: true
```

---

When we have gnosis safe payouts integrated the `payment-permit-generation` will start to seem less clear than the `auto-pay-mode` naming

---

I think the more granular configs we have, the better for enterprise/business users (our partners.)

I thought the vision was to have an `auto-pay-up-to-price` property, which will automatically post a payment permit if the bounty is below a certain price (perhaps that isn't the best property name). 

We could make this a "required config" (similar to TypeScript with codependent configs) if SAFE is included.

### Scenarios

1. Without automatic payments (payment permits) and without SAFE 
```
payment-permit-generation: false
# safe-address: 0xb18D844f3578EaD22791B5c67F7B6aF93266804E
# auto-pay-up-to-price: 100
```
2. With automatic payments (payment permits) and without SAFE 
```
payment-permit-generation: true
# safe-address: 0xb18D844f3578EaD22791B5c67F7B6aF93266804E
auto-pay-up-to-price: 100
```
3. Without automatic payments (payment permits) and with SAFE 
```
payment-permit-generation: false
safe-address: 0xb18D844f3578EaD22791B5c67F7B6aF93266804E
# auto-pay-up-to-price: 100
```
4. With automatic payments (payment permits) and with SAFE 
```
payment-permit-generation: true
safe-address: 0xb18D844f3578EaD22791B5c67F7B6aF93266804E
auto-pay-up-to-price: 100
```